pkg/daemon/update.go

```diff
@@ -1988,6 +1990,77 @@ func (dn *Daemon) cancelSIGTERM() {
 	}
 }
 
+func (dn *Daemon) SetLabelsOnNode(labels map[string]interface{}) error {
```

Comment:
why exposed?

Reply:
+1 Also it's not setting _all_ labels on Node (as name indictes) it's only setting the LB labels on node.